keps/sig-auth/20190607-certificates-api.md

```diff
@@ -292,8 +293,30 @@ type CertificateSigningRequestStatus struct {
   // Conditions applied to the request, such as approval or denial.
   Conditions []CertificateSigningRequestCondition
 
-  // If request was approved, the controller will place the issued
-  // certificate here.
+  // Certificate is populated by the signer with the issued certificate in PEM format.
+  // In JSON and YAML output, this entire field is base64-encoded, so it consists of:
+  // base64(
+  // -----BEGIN CERTIFICATE-----
+  // MII...Pb7Yu/E=
+  // -----END CERTIFICATE-----
+  // optional intermediate certificate blocks
+  // -----BEGIN CERTIFICATE-----
+  // MII...MGKB
+  // -----END CERTIFICATE-----
+  // -----BEGIN CERTIFICATE-----
+  // MII...AY1M
+  // -----END CERTIFICATE-----
+  // )
+  //
+  // In v1beta1, this field is unvalidated.
```

Comment:
Is there any risk of a v1 (machine) client getting stuck trying to update a CSR that is not valid per the v1 format (i.e. it was created via the v1beta1 API)?

Reply:
updates that did not modify that field value would be allowed; will clarify